.changelog/2707.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+api-gateway: adds 2000 to any privileged port defined on gateway listeners
```

Comment:
this might need some help

Reply:
```suggestion
api-gateway: map privileged ports on Gateway listeners to unprivileged ports so that containers don't require additional privileges 
```